test/visualTest.ts

```diff
@@ -574,20 +574,72 @@ describe("EnhancedScatterChart", () => {
+                    expect(element.style.strokeWidth).not.toBe("0px");
                 });
 
                 (<any>dataView.metadata.objects).outline.show = false;
                 visualBuilder.updateFlushAllD3Transitions(dataView);
 
                 visualBuilder.dots.forEach((element: HTMLElement) => {
-                    assertColorsMatch(element.style.fill, element.style.stroke);
+                    expect(element.style.strokeWidth).toBe("0px");
                 });
             });
+
+            it("stroke width", () => {
+                const strokeWidth: number = getRandomNumber(1, 5);
+
+                (<any>dataView.metadata.objects).outline.strokeWidth = strokeWidth;
+
+                visualBuilder.updateFlushAllD3Transitions(dataView);
+
+                visualBuilder.dots.forEach((element: HTMLElement) => {
+                    expect(element.style.strokeWidth).toBe(`${strokeWidth}px`);
+                });
+            });
+
+            it("stroke before and after apply of high-contrast", () => {
+                visualBuilder.updateFlushAllD3Transitions(dataView);
+
+                visualBuilder.dots.forEach((element: HTMLElement) => {
+                    assertColorsMatch(element.style.fill, element.style.stroke, true)
+                });
+
+                const backgroundColor: string = "#000000";
+                const foregroundColor: string = "#ffff00";
+    
+                visualBuilder.visualHost.colorPalette.isHighContrast = true;
+                visualBuilder.visualHost.colorPalette.background = { value: backgroundColor };
+                visualBuilder.visualHost.colorPalette.foreground = { value: foregroundColor };
+
+                visualBuilder.updateFlushAllD3Transitions(dataView);
+
+                visualBuilder.dots.forEach((element: HTMLElement) => {
+                    assertColorsMatch(element.style.fill, element.style.stroke, true)
+                });
+            });
+
+            it("focus-visible state during keyboard navigation", () => {
+                const focusedElementOutlineStrokeWidth: number = 8;
+                visualBuilder.updateFlushAllD3Transitions(dataView);
+
+                const firstElement: HTMLElement = visualBuilder.dots[0];
+                firstElement.focus();
+
+                const computedStyles = getComputedStyle(firstElement);
+                const stroke = computedStyles.getPropertyValue("stroke");
+                const strokeWidth = computedStyles.getPropertyValue("stroke-width");
+                const fill = computedStyles.getPropertyValue("fill");
+
+                expect(strokeWidth).toBe(`${focusedElementOutlineStrokeWidth}px`);
+                assertColorsMatch(fill, stroke, true);
+
+                for(let index = 1; index < visualBuilder.dots.length; index++){
+                    expect(strokeWidth).not.toBe(visualBuilder.dots[index].style.strokeWidth)
```

Comment:
Focused element should always have a greater stroke width than the others. Try something like
expect(strokeWidth).toBeGreaterThan(visualBuilder.dots[index].style.strokeWidth)

Reply:
resolved here https://github.com/microsoft/powerbi-visuals-enhancedscatter/pull/102/commits/465294c3359a377c23bede03e11378a748c01c4a